.cirrus.yml

```diff
@@ -452,6 +450,21 @@ task:
     COCOAPODS_DISABLE_STATS: true
     CPU: 2
     MEMORY: 8G
+  all_flutter_cache:
+    folder: bin/cache/
```

Comment:
@Hixie @dnfield Do either of you know why the other shards have separate Cirrus caches for `pkg` and `artifacts` in the `bin/cache` but not that entire directory?  I don't think the Linux or Windows caches are getting used since all the stamps are missing and it just re-downloads everything again.
Example:
https://cirrus-ci.com/task/6607166220009472?command=setup#L420

Reply:
I think that's just how they were always set up - it does look like we're getting cache hits (https://cirrus-ci.com/task/6607166220009472?command=flutter_pkg#L16) but maybe failing to use them in setup...

It seems likely to me that this worked at some point and we fixed something in the tool that broke it.

Reply:
Yeah, even worse than ignoring the cache, we're downloading it, then the tool blows it away because the stamp files aren't there and re-downloads them. Should this PR include linux + windows?

Reply:
I was going to do Windows and Linux in a separate PR.  I want to get the mac changes out the door ASAP with Cirrus's network flakiness.